packages/node_modules/@webex/plugin-meetings/test/unit/spec/peerconnection-manager/index.js

```diff
@@ -0,0 +1,43 @@
+
+import {assert} from '@webex/test-helper-chai';
+import sinon from 'sinon';
+import PeerConnectionManager from '@webex/plugin-meetings/src/peer-connection-manager/index';
+import StaticConfig from '@webex/plugin-meetings/src/common/config';
+
+describe('Peerconnection Manager', () => {
+  beforeEach(() => {
+    StaticConfig.set({bandwidth: {audio: 50, video: 500, startBitrate: 2000}});
+  });
+  describe('Methods', () => {
+    describe('setRemoteSessionDetails', () => {
+      it('change the start bitrate on remoteSDP', async () => {
```

Comment:
is there also a test that checks that "x-google-start-bitrate" is not added when startBitrate is undefined in config?

Reply:
done